src/service/fixed-layer.js

```diff
@@ -306,37 +323,82 @@ export class FixedLayer {
   }
 
   /**
-   * Apply or reset transform style to fixed elements. The existing transition,
-   * if any, is disabled when custom transform is supplied.
-   * @param {?string} transform
+   * Apply transform style to all fixed elements, unless specified.
+   *
+   * Ignores `independent` elements when a transform is specified but an
+   * element is not. Resets all when neither is specified.
+   *
+   * @param {string=} opt_addedTransform undefined to reset.
+   * @param {!Element=} opt_element undefined to go through all.
    */
-  transformMutate(transform) {
-    // Unfortunately, we can't do anything with sticky elements here. Updating
-    // `top` in animation frames causes reflow on all platforms and we can't
-    // determine whether an element is currently docked to apply transform.
-    if (transform) {
-      // Apply transform style to all fixed elements
-      this.elements_.forEach(e => {
-        if (e.fixedNow && e.top) {
-          setStyle(e.element, 'transition', 'none');
-          if (e.transform && e.transform != 'none') {
-            setStyle(e.element, 'transform', e.transform + ' ' + transform);
-          } else {
-            setStyle(e.element, 'transform', transform);
-          }
-        }
-      });
-    } else {
-      // Reset transform style to all fixed elements
```

Comment:
How do we reset the transforms now?

Reply:
Within the same loop. `combineTransforms` knows what to do.